[ObsUX][Otel Hosts] Schema selection

MiriamAparicio opened the pull request:
Closes https://github.com/elastic/kibana/issues/226335

### Summary

A select to Hosts page is added to let users select preferred schema available

#### What has been done

- The select is always present, even when there's just one schema available
- The schema selector should show how many available schemas are available based on the current query
- When loading the app, the schema selector should default to ECS (if only ECS available), OTel (if only OTel available) or OTel (when both are available)
- Once a schema is selected, the preferred schema is persisted in the url params
- If the user changes the query and the currently selected schema is not available anymore, we should show the selected schema is invalid
- Update host_count api to take the preferred selected schema
- Make sure the new select component follows a11y

#### How to test

- This feature is under a feature flag, set in kibana.yml config the `xpack.infra.featureFlags.hostsOtelEnabled` to `true`
- Use https://github.com/smith/elastic-stack-docker-compose?tab=readme-ov-file#docker-compose-metrics to get Otel metrics

https://github.com/user-attachments/assets/3f48ed1d-b7cc-4cdb-95b8-ddb52d999af5

MiriamAparicio committed:
fix service groups button focus

MiriamAparicio committed:
add schema selection to hosts page

MiriamAparicio committed:
use semconv as default if both schemas exist

MiriamAparicio committed:
set preferred schema in url

MiriamAparicio committed:
fix issues, reimplement schema selector

MiriamAparicio committed:
add feature flag and a11y improvements

MiriamAparicio committed:
fix linting error

crespocarlos reviewed (commented):
Thanks for this @MiriamAparicio . I haven't tested it yet but wanted to review it already. We could already use the Schema Selector state in the `useHostCount` hook

MiriamAparicio committed:
rollback not related changes with this PR

MiriamAparicio commented on x-pack/solutions/observability/plugins/infra/public/pages/metrics/hosts/hooks/use_unified_search.ts:
```diff
@@ -28,15 +29,28 @@ import { retrieveFieldsFromFilter } from '../../../../utils/filters/build';
     limit,
   };
+
+  if (typeof preferredSchema === 'string') {
```
preferredSchema can be initially null, so we don't want/need to send that to telemetry, if preferredSchema is null we don't add it to the submitted payload

rmyz replied:
Why don't we check if it's not falsy instead?

MiriamAparicio commented on x-pack/solutions/observability/plugins/infra/public/components/schema_selector.tsx:
```diff
@@ -0,0 +1,183 @@
+                  options={displayOptions}
+                  valueOfSelected={value === null ? '' : value}
+                  onChange={onSelect}
+                  isLoading={!schemas.length}
```
Good catch, it's leftover when I was testing the loading state on the SuperSelect component, I will add a proper loading state

MiriamAparicio committed:
pr comments & fix anomaly e2e test

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/322461)
* Commit: e1dd6b43a7360719953fc79104f51fefc06d3519

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/322461#019837af-879c-423c-97cf-36a6b71b1a64)

### History
* :broken_heart: [Build #322306](https://buildkite.com/elastic/kibana-pull-request/builds/322306) failed 5e8091aecbd156c4172d4d6296f19f213c72c81f
* :broken_heart: [Build #322226](https://buildkite.com/elastic/kibana-pull-request/builds/322226) failed b77f6f99755f691a3dd37000ab1d461ffd025449

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322461","number":322461,"commit":"e1dd6b43a7360719953fc79104f51fefc06d3519"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322306","number":322306,"commit":"5e8091aecbd156c4172d4d6296f19f213c72c81f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322226","number":322226,"commit":"b77f6f99755f691a3dd37000ab1d461ffd025449"}],"number":322461}
buildkite-pr-comment-->

MiriamAparicio committed:
fix constant imports

MiriamAparicio committed:
update host_count to take preferred selected schema

MiriamAparicio committed:
Merge remote-tracking branch 'upstream/main' into 226335-schema-selection

MiriamAparicio committed:
rollback not related changes with this PR

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/322502)
* Commit: 589b1a654c2f4dc369df46b2d62afc33efddf9b2

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/322502#019837e4-416f-4f9a-adca-a77a20fb113c)

### History
* :broken_heart: [Build #322482](https://buildkite.com/elastic/kibana-pull-request/builds/322482) failed 9d77ed314a92e39f62307e2bae5d41a291cc8ec4
* :broken_heart: [Build #322461](https://buildkite.com/elastic/kibana-pull-request/builds/322461) failed e1dd6b43a7360719953fc79104f51fefc06d3519
* :broken_heart: [Build #322306](https://buildkite.com/elastic/kibana-pull-request/builds/322306) failed 5e8091aecbd156c4172d4d6296f19f213c72c81f
* :broken_heart: [Build #322226](https://buildkite.com/elastic/kibana-pull-request/builds/322226) failed b77f6f99755f691a3dd37000ab1d461ffd025449

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322502","number":322502,"commit":"589b1a654c2f4dc369df46b2d62afc33efddf9b2"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322482","number":322482,"commit":"9d77ed314a92e39f62307e2bae5d41a291cc8ec4"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322461","number":322461,"commit":"e1dd6b43a7360719953fc79104f51fefc06d3519"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322306","number":322306,"commit":"5e8091aecbd156c4172d4d6296f19f213c72c81f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322226","number":322226,"commit":"b77f6f99755f691a3dd37000ab1d461ffd025449"}],"number":322502}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/322524)
* Commit: 5e5a121b1fc4ac43a01b4eb367576d8e7c0bfabb

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/322524#01983800-311a-4baf-bc9f-2918bb7e756c)

### History
* :broken_heart: [Build #322502](https://buildkite.com/elastic/kibana-pull-request/builds/322502) failed 589b1a654c2f4dc369df46b2d62afc33efddf9b2
* :broken_heart: [Build #322482](https://buildkite.com/elastic/kibana-pull-request/builds/322482) failed 9d77ed314a92e39f62307e2bae5d41a291cc8ec4
* :broken_heart: [Build #322461](https://buildkite.com/elastic/kibana-pull-request/builds/322461) failed e1dd6b43a7360719953fc79104f51fefc06d3519
* :broken_heart: [Build #322306](https://buildkite.com/elastic/kibana-pull-request/builds/322306) failed 5e8091aecbd156c4172d4d6296f19f213c72c81f
* :broken_heart: [Build #322226](https://buildkite.com/elastic/kibana-pull-request/builds/322226) failed b77f6f99755f691a3dd37000ab1d461ffd025449

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322524","number":322524,"commit":"5e5a121b1fc4ac43a01b4eb367576d8e7c0bfabb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322502","number":322502,"commit":"589b1a654c2f4dc369df46b2d62afc33efddf9b2"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322482","number":322482,"commit":"9d77ed314a92e39f62307e2bae5d41a291cc8ec4"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322461","number":322461,"commit":"e1dd6b43a7360719953fc79104f51fefc06d3519"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322306","number":322306,"commit":"5e8091aecbd156c4172d4d6296f19f213c72c81f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322226","number":322226,"commit":"b77f6f99755f691a3dd37000ab1d461ffd025449"}],"number":322524}
buildkite-pr-comment-->

crespocarlos commented on x-pack/solutions/observability/plugins/infra/public/pages/metrics/hosts/components/search_bar/unified_search_bar.tsx:
```diff
@@ -40,9 +71,21 @@ export const UnifiedSearchBar = () => {
     [onSubmit, onPageRefreshStart]
   );
 
+  const isLoading = status === FETCH_STATUS.LOADING || status === FETCH_STATUS.PENDING;
```
```suggestion
  const isLoading = isPending(status);
```

MiriamAparicio commented on x-pack/solutions/observability/plugins/observability_shared/common/locators/infra/hosts_locator.ts:
```diff
@@ -23,6 +23,7 @@ export interface HostsLocatorParams extends SerializableRecord {
   filters?: Filter[];
   panelFilters?: Filter[];
   limit?: number;
+  preferredSchema?: 'ecs' | 'semconv' | null;
```
I can not import the type there, I can create it, but not imported

MiriamAparicio committed:
pr review

MiriamAparicio committed:
Merge branch 'main' into 226335-schema-selection

tonyghiani reviewed (approved):
I'll defer the review to members of the infra team as we the logs team only shares ownership on the locator files but there are no changes affecting our team, approving to unblock.

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/lint_ts_projects --fix'

maximpn committed:
[Security Solution] Implement prebuilt rule upgrade notifications test plan (#225912)

**Addresses:** https://github.com/elastic/kibana/issues/202078

## Summary

This PR implements the [Prebuilt Rules Upgrade Notifications test plan](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rule_upgrade_notifications.md).

## Details

This PR implements test scenarios listed in the [Prebuilt Rules Upgrade Notifications test plan](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rule_upgrade_notifications.md). Unit tests listed in the test plan was skipped due to low testability of the UI code, high maintenance effort and low total value since the same scenarios are covered by the e2e tests.

## Flaky test runner

- ✅ [added integration tests (200 runs)](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/8656)
- ✅ [added e2e tests (75 runs)](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/8664) Since the whole Rule Management Cypress tests group was ran you may observe some failures in unrelated to this PR tests.

bartoval committed:
[ES|QL]: Integrate Elasticsearch Command Metadata into ESQL Registry (#229172)

## Summary

This PR is part of https://github.com/elastic/kibana/issues/216791 and a
prequel of https://github.com/elastic/kibana/pull/228897

- new script to generate metadata for commands, from elasticsearch
- compatibility script to generate functions and commands using the
usual syntax yarn make:defs <path of elasticsearch>
- adds information to the command registry
- Test the logic

<img width="1325" height="572" alt="pr"
src="https://github.com/user-attachments/assets/fae46153-c294-421b-9c4c-aa105de4bb9f"
/>

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/322734)
* Commit: 58ed02c1d5bc777c79e6fcb26dbd5a6d902ce7d0

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/322734#01983be2-b3ac-4cb7-b45f-ddf1efec3afe)

### History
* :broken_heart: [Build #322536](https://buildkite.com/elastic/kibana-pull-request/builds/322536) failed 5a7d04b794bf8950f62baf81e1affb4b1c96c963
* :broken_heart: [Build #322524](https://buildkite.com/elastic/kibana-pull-request/builds/322524) failed 5e5a121b1fc4ac43a01b4eb367576d8e7c0bfabb
* :broken_heart: [Build #322502](https://buildkite.com/elastic/kibana-pull-request/builds/322502) failed 589b1a654c2f4dc369df46b2d62afc33efddf9b2
* :broken_heart: [Build #322482](https://buildkite.com/elastic/kibana-pull-request/builds/322482) failed 9d77ed314a92e39f62307e2bae5d41a291cc8ec4
* :broken_heart: [Build #322461](https://buildkite.com/elastic/kibana-pull-request/builds/322461) failed e1dd6b43a7360719953fc79104f51fefc06d3519

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322734","number":322734,"commit":"58ed02c1d5bc777c79e6fcb26dbd5a6d902ce7d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322536","number":322536,"commit":"5a7d04b794bf8950f62baf81e1affb4b1c96c963"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322524","number":322524,"commit":"5e5a121b1fc4ac43a01b4eb367576d8e7c0bfabb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322502","number":322502,"commit":"589b1a654c2f4dc369df46b2d62afc33efddf9b2"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322482","number":322482,"commit":"9d77ed314a92e39f62307e2bae5d41a291cc8ec4"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322461","number":322461,"commit":"e1dd6b43a7360719953fc79104f51fefc06d3519"}],"number":322734}
buildkite-pr-comment-->

MiriamAparicio committed:
Merge remote-tracking branch 'upstream/main' into 226335-schema-selection

MiriamAparicio committed:
fix circular dependency

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/322765)
* Commit: fb279e82cce0854e4e7023cc0156c0af2d8e3406

### Failed CI Steps
* [Jest Tests #1](https://buildkite.com/elastic/kibana-pull-request/builds/322765#01983c2e-9825-47f8-9a3a-45e4def9128c)
* [FTR Configs #18](https://buildkite.com/elastic/kibana-pull-request/builds/322765#01983c2e-983a-4785-9363-ee459f58918b)
* [FTR Configs #18](https://buildkite.com/elastic/kibana-pull-request/builds/322765#01983c50-72c1-4e3a-a15a-2ef28fa288a9)
* [FTR Configs #22](https://buildkite.com/elastic/kibana-pull-request/builds/322765#01983c2e-983c-4d51-bbd1-f0e77d39e24f)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322765#01983c2e-983a-4785-9363-ee459f58918b) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322765/jobs/01983c2e-983a-4785-9363-ee459f58918b/artifacts/01983c50-591f-448f-ab79-0d57fd96b0ca) FTR Configs #<span></span>18 / APM API tests fleet/apm_package_policy.spec.ts basic no archive APM package policy "after all" hook in "APM package policy"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322765#01983c50-72c1-4e3a-a15a-2ef28fa288a9) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322765/jobs/01983c50-72c1-4e3a-a15a-2ef28fa288a9/artifacts/01983c57-b87a-4517-8740-883918d8851d) FTR Configs #<span></span>18 / APM API tests fleet/apm_package_policy.spec.ts basic no archive APM package policy "after all" hook in "APM package policy"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322765#01983c2e-983a-4785-9363-ee459f58918b) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322765/jobs/01983c2e-983a-4785-9363-ee459f58918b/artifacts/01983c50-591e-4ccc-830f-553b558a4850) FTR Configs #<span></span>18 / APM API tests fleet/apm_package_policy.spec.ts basic no archive APM package policy "before all" hook in "APM package policy"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322765#01983c50-72c1-4e3a-a15a-2ef28fa288a9) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322765/jobs/01983c50-72c1-4e3a-a15a-2ef28fa288a9/artifacts/01983c57-b87a-4d24-ac16-f4908a2586b3) FTR Configs #<span></span>18 / APM API tests fleet/apm_package_policy.spec.ts basic no archive APM package policy "before all" hook in "APM package policy"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322765#01983c2e-983c-4d51-bbd1-f0e77d39e24f) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322765/jobs/01983c2e-983c-4d51-bbd1-f0e77d39e24f/artifacts/01983c56-5d5c-4263-b6f1-96a3a431dfb7) FTR Configs #<span></span>22 / APM API tests fleet/migration_check.spec.ts cloud no archive Fleet migration check - cloud has_cloud_agent_policy with Cloud agent policy "before all" hook for "should be true when cloud agent policy exists"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322765#01983c2e-9825-47f8-9a3a-45e4def9128c) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322765/jobs/01983c2e-9825-47f8-9a3a-45e4def9128c/artifacts/01983c54-6c02-4d54-a3ff-62581a1facdf) Jest Tests #<span></span>1 / useHostCount when data is fetched with error should NOT call reportHostsViewTotalHostCountRetrieved 

### History
* :broken_heart: [Build #322734](https://buildkite.com/elastic/kibana-pull-request/builds/322734) failed 58ed02c1d5bc777c79e6fcb26dbd5a6d902ce7d0
* :broken_heart: [Build #322536](https://buildkite.com/elastic/kibana-pull-request/builds/322536) failed 5a7d04b794bf8950f62baf81e1affb4b1c96c963
* :broken_heart: [Build #322524](https://buildkite.com/elastic/kibana-pull-request/builds/322524) failed 5e5a121b1fc4ac43a01b4eb367576d8e7c0bfabb
* :broken_heart: [Build #322502](https://buildkite.com/elastic/kibana-pull-request/builds/322502) failed 589b1a654c2f4dc369df46b2d62afc33efddf9b2
* :broken_heart: [Build #322482](https://buildkite.com/elastic/kibana-pull-request/builds/322482) failed 9d77ed314a92e39f62307e2bae5d41a291cc8ec4
* :broken_heart: [Build #322461](https://buildkite.com/elastic/kibana-pull-request/builds/322461) failed e1dd6b43a7360719953fc79104f51fefc06d3519

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322765","number":322765,"commit":"fb279e82cce0854e4e7023cc0156c0af2d8e3406"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322734","number":322734,"commit":"58ed02c1d5bc777c79e6fcb26dbd5a6d902ce7d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322536","number":322536,"commit":"5a7d04b794bf8950f62baf81e1affb4b1c96c963"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322524","number":322524,"commit":"5e5a121b1fc4ac43a01b4eb367576d8e7c0bfabb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322502","number":322502,"commit":"589b1a654c2f4dc369df46b2d62afc33efddf9b2"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322482","number":322482,"commit":"9d77ed314a92e39f62307e2bae5d41a291cc8ec4"}],"number":322765}
buildkite-pr-comment-->

MiriamAparicio committed:
add a fallback for searchCriteria in use_host_count

MiriamAparicio committed:
Merge branch 'main' into 226335-schema-selection

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint_all_files --no-cache --fix'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/322906)
* Commit: 927a8b32e510fb8785d66649623d795ea66f0c29

### Failed CI Steps
* [FTR Configs #34](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983ce5-3918-469c-963d-25fa30ec5605)
* [FTR Configs #37](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983ce5-391a-4530-bc6a-5925155e947f)
* [FTR Configs #37](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983cfe-48d2-4c11-a316-a002ee9446f1)
* [FTR Configs #54](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983ce5-3925-417e-8ed6-ac45711d8227)
* [FTR Configs #64](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983ce5-392e-4e27-ae69-2d16edb4b40d)
* [FTR Configs #78](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983ce5-393b-4c07-8c8a-cdc7a24a2f16)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983ce5-3918-469c-963d-25fa30ec5605) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322906/jobs/01983ce5-3918-469c-963d-25fa30ec5605/artifacts/01983d0c-163e-4b9f-905b-9b354f4c7749) FTR Configs #<span></span>34 / APM API tests fleet/apm_package_policy.spec.ts basic no archive APM package policy "after all" hook in "APM package policy"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983ce5-3918-469c-963d-25fa30ec5605) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322906/jobs/01983ce5-3918-469c-963d-25fa30ec5605/artifacts/01983d0c-163e-4333-ac11-cce82dfd4194) FTR Configs #<span></span>34 / APM API tests fleet/apm_package_policy.spec.ts basic no archive APM package policy "before all" hook in "APM package policy"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983ce5-3925-417e-8ed6-ac45711d8227) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322906/jobs/01983ce5-3925-417e-8ed6-ac45711d8227/artifacts/01983d0b-2d0b-4469-83ae-22e5fd813ebf) FTR Configs #<span></span>54 / APM API tests fleet/migration_check.spec.ts cloud no archive Fleet migration check - cloud has_cloud_agent_policy with Cloud agent policy "before all" hook for "should be true when cloud agent policy exists"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983ce5-391a-4530-bc6a-5925155e947f) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322906/jobs/01983ce5-391a-4530-bc6a-5925155e947f/artifacts/01983cfe-2f0c-409d-8983-47e2bbc2a57c) FTR Configs #<span></span>37 / Cloud Security Posture Test adding Cloud Security Posture Integrations CNVM CNVM AWS Hyperlink on PostInstallation Modal should have the correct URL
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983cfe-48d2-4c11-a316-a002ee9446f1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322906/jobs/01983cfe-48d2-4c11-a316-a002ee9446f1/artifacts/01983d0e-ca7e-43ab-8273-1b3174c545a3) FTR Configs #<span></span>37 / Cloud Security Posture Test adding Cloud Security Posture Integrations CNVM CNVM AWS Hyperlink on PostInstallation Modal should have the correct URL
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983ce5-393b-4c07-8c8a-cdc7a24a2f16) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322906/jobs/01983ce5-393b-4c07-8c8a-cdc7a24a2f16/artifacts/01983cfd-1b4a-4fd5-a0cf-18b9f6c9deb9) FTR Configs #<span></span>78 / serverless observability UI Onboarding Onboarding Auto-Detect guides user through data onboarding
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/322906#01983ce5-392e-4e27-ae69-2d16edb4b40d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/322906/jobs/01983ce5-392e-4e27-ae69-2d16edb4b40d/artifacts/01983d0b-3cfb-4929-93c1-834f8a2d4ca1) FTR Configs #<span></span>64 / Synthetics API Tests AddProjectMonitors "before all" hook for "project monitors - saves space as data stream namespace"

### History
* :broken_heart: [Build #322765](https://buildkite.com/elastic/kibana-pull-request/builds/322765) failed fb279e82cce0854e4e7023cc0156c0af2d8e3406
* :broken_heart: [Build #322734](https://buildkite.com/elastic/kibana-pull-request/builds/322734) failed 58ed02c1d5bc777c79e6fcb26dbd5a6d902ce7d0
* :broken_heart: [Build #322536](https://buildkite.com/elastic/kibana-pull-request/builds/322536) failed 5a7d04b794bf8950f62baf81e1affb4b1c96c963
* :broken_heart: [Build #322524](https://buildkite.com/elastic/kibana-pull-request/builds/322524) failed 5e5a121b1fc4ac43a01b4eb367576d8e7c0bfabb
* :broken_heart: [Build #322502](https://buildkite.com/elastic/kibana-pull-request/builds/322502) failed 589b1a654c2f4dc369df46b2d62afc33efddf9b2

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322906","number":322906,"commit":"927a8b32e510fb8785d66649623d795ea66f0c29"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322765","number":322765,"commit":"fb279e82cce0854e4e7023cc0156c0af2d8e3406"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322734","number":322734,"commit":"58ed02c1d5bc777c79e6fcb26dbd5a6d902ce7d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322536","number":322536,"commit":"5a7d04b794bf8950f62baf81e1affb4b1c96c963"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322524","number":322524,"commit":"5e5a121b1fc4ac43a01b4eb367576d8e7c0bfabb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322502","number":322502,"commit":"589b1a654c2f4dc369df46b2d62afc33efddf9b2"}],"number":322906}
buildkite-pr-comment-->

MiriamAparicio committed:
Merge branch 'main' into 226335-schema-selection
